RestrictEvents/RestrictEvents.cpp

```diff
@@ -245,8 +246,14 @@ struct RestrictEventsPolicy {
 		CPUInfo::getCpuid(0, 0, nullptr, &b, &c, &d);
 		bool isAMD = (b == CPUInfo::signature_AMD_ebx && c == CPUInfo::signature_AMD_ecx && d == CPUInfo::signature_AMD_edx);
 
-		pmKextRegister(PM_DISPATCH_VERSION, NULL, &pmCallbacks);
 		uint32_t cc = 0, pp = 0;
+		if (!isAMD) {
+			cc = cpuid_info()->core_count;
```

Comment:
`cpuid_info()` is `com.apple.kpi.private`, but I guess we cannot get there via any other way.